pkg/netid/netid.go

```diff
@@ -0,0 +1,61 @@
+package netid
+
+/*
+Accessor methods to annotate namespace for multitenant support
+*/
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+const (
+	// Current assigned VNID for the namespace
+	VNIDAnnotation string = "pod.network.openshift.io/multitenant.vnid"
+	// Desired VNID for the namespace
+	WantsVNIDAnnotation string = "pod.network.openshift.io/multitenant.wants-vnid"
```

Comment:
Quick question - why is there a "wants-vnid" and "vnid"?  When is wants != vnid?